main/ui/src/main/java/org/cryptomator/ui/vaultoptions/MountOptionsController.java

```diff
@@ -52,13 +52,20 @@
 	public RadioButton mountPointCustomDir;
 	public ChoiceBox<String> driveLetterSelection;
 
+	//FUSE + Windows -> Disable some (experimental) features for the user because they are unstable
+	//Use argument Dfuse.experimental="true" to override
+	private final BooleanBinding fuseAndWindows;
+
 	@Inject
 	MountOptionsController(@VaultOptionsWindow Stage window, @VaultOptionsWindow Vault vault, Settings settings, WindowsDriveLetters windowsDriveLetters, ResourceBundle resourceBundle) {
 		this.window = window;
 		this.vault = vault;
 		this.webDavAndWindows = settings.preferredVolumeImpl().isEqualTo(VolumeImpl.WEBDAV).and(osIsWindows);
 		this.windowsDriveLetters = windowsDriveLetters;
 		this.resourceBundle = resourceBundle;
+
+		System.out.println(Boolean.getBoolean("fuse.experimental"));
+		this.fuseAndWindows = settings.preferredVolumeImpl().isEqualTo(VolumeImpl.FUSE).and(osIsWindows).and(new SimpleBooleanProperty(!Boolean.getBoolean("fuse.experimental")));
```

Comment:
Instead of `Boolean.getBoolean("fuse.experimental")`, use our `Environment` singleton to access system properties, so we have all potentially relevant properties in one place (easier to maintain and document).

Reply:
Resolved by f45b78d8c060042ec444f92254e1e8707b1ad72e